tests/webgpu_basic_rendering.cpp

```diff
@@ -27,7 +27,7 @@ void GetDevice(void (*callback)(wgpu::Device)) {
         }
         if (status == WGPURequestAdapterStatus_Unavailable) {
             printf("WebGPU unavailable; exiting cleanly\n");
-            exit(0);
+            return;
```

Comment:
Why not `exit(0)` here?

Reply:
This helps test that the runtime-keepalive is working correctly. If the runtime doesn't exit implicitly after this return, then the test times out. Adding a comment.